src/components/panel/demoGroups/index.html

```diff
@@ -0,0 +1,46 @@
+<div ng-controller="PanelGroupsCtrl as ctrl" ng-cloak>
+
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <md-button
+        class="md-icon-button"
+        aria-label="Settings"
+        ng-click="ctrl.showMenu($event, ctrl.settings, 'toolbar')">
+        <md-icon md-svg-icon="img/icons/menu.svg"></md-icon>
+      </md-button>
+      <h2>Toolbar with grouped panels (Maximum open: 2)</h2>
+      <span flex></span>
+      <md-button
+        class="md-icon-button"
+        aria-label="Favorite"
+        ng-click="ctrl.showMenu($event, ctrl.favorite, 'toolbar')">
```

Comment:
@ErinCoughlan I can't do that as I am accessing `menu.items` from the passed-in variable on line 70.